Create app UI

KokkieH opened the pull request:
Completes the markup task in #1 

This PR does the following:

* Adds media files in the app `/src` folder
* Creates the HTML markup, including an interactive image that will serve as the clickable element to display a new quote (for now clicking it merely displays a notice in the browser)
* Adds CSS styling.

## Testing instructions

- Check out this branch on your local environment
- In the folder, `/random-quote-spa/random-quote-generator` run `npm install`
- run `npm start`
- View the app at http://localhost:3000/
- The app should appear like this (for now it will only display a static quote):

![Random_Quote_Generator_and_App_js_—_random-quote-spa](https://user-images.githubusercontent.com/11873759/159269969-f81f9f70-17d3-4aa6-af16-fb6fd4cc34f4.png)

- The "inkpot" button should dip the quill on hover:

![Random_Quote_Generator](https://user-images.githubusercontent.com/11873759/159272014-b4cf8ed6-5710-4bcf-885c-3982a11a8e95.png)

- Also make sure the button is clickable. When clicked, it should display a notice in the browser like this:
![Random_Quote_Generator](https://user-images.githubusercontent.com/11873759/159270470-d85f0662-f7ad-4ad5-a5be-50c5a82d5e7b.png)

KokkieH committed:
Add app media files
Add markup using JSX
Add CSS

KokkieH committed:
Make inkpot image clickable

KokkieH committed:
Change HTML title

KokkieH committed:
Add static quote text for testing purposes

tgglv commented on random-quote-generator/src/App.js:
```diff
@@ -1,24 +1,30 @@
+          <p id="author">- William Shakespeare</p>
+          <img id="button" src={quill} alt="new quote button" 
+          onMouseOver={e => (e.currentTarget.src=quill2)}
+          onMouseOut={e => (e.currentTarget.src=quill)}
```
Hey Herman 👋 
Thanks for this implementation, it's definitely a great start.

To let this code look more React-ish, you could try this:
```js
const [ imageSrc, setImageSrc ] = useState( quill ); // at the top of the App function
...
return (
    ...
    <img id="button" src={ imageSrc } ...
        onMouseOver={ () => setImageSrc( quill2 ) }
        onMouseOut=( () => setImageSrc( quill ) }
        ...
```

KokkieH replied:
Thanks. I didn't look at your code, wanting to figure it out on my own first, but ended up with basically the same thing :)

I also added a static array with a few quotes for now, and changing the state to let the quotes update when clicking the button.

tgglv reviewed (approved):
Thanks, @KokkieH!

The code you wrote looks good! Left one comment to make it even better.
Feel free to merge this code into the main branch.

KokkieH committed:
use React state to change inkpot button

KokkieH committed:
Add quotes

Add static array of quotes
Display a random quote using React state
Change quote being displayed when button is clicked
